specification/app/Microsoft.App/models/code-execution.tsp

```diff
@@ -36,25 +36,25 @@ union CodeInputType {
   string,
 
   @doc("Inline.")
-  inline: "inline",
+  inline: "Inline",
 
   @doc("InlineBase64.")
-  inlineBase64: "inlineBase64",
+  inlineBase64: "InlineBase64",
 
   @doc("InlineText.")
-  inlineText: "inlineText",
+  inlineText: "InlineText",
 }
 
 @doc("Execution Type")
 union ExecutionType {
   string,
 
   @doc("Synchronous")
-  synchronous: "synchronous",
+  synchronous: "Synchronous",
```

Comment:
we should also add type 'Asynchronous' as recently we added support for that. 

enableEgress & enableMultiArray fields are also missing.